### PR TITLE
Add --plot-to-board to plot command

### LINKED ISSUE #134: Add --plot-to-board to plot command
This option would be also useful in this command. This will send the plotted burndown chart to the first card of the `Done` column.

### PR DESCRIPTION
**Pull request by Ronaq13**
This flag of plot command will send the plotted burndown chart to the first
card of the `Done` column.

close #134

### CODE REVIEW

File: lib/cli.rb [hunk 1]
```diff
@@ -189,13 +189,24 @@ def burndown
...
   option 'with-fast-lane', desc: 'Plot Fast Lane with new cards bars', required: false, type: :boolean
   option 'no-tasks', desc: 'Do not plot tasks line', required: false, type: :boolean
+  option :plot_to_board, type: :boolean, desc: 'Send the plotted data to the first card of Done column'
   def plot(sprint_number)
     process_global_options options
-    BurndownChart.plot(sprint_number, options)
+    if options[:plot_to_board]
+      begin
+        chart = BurndownChart.new @@settings
+        chart.update(options)
```
Comment (Ana06) [inline lib/cli.rb]: This is also modifying the `yml` file, which we don't want to modify here. We only want to print the image and push it to the card.

**Review by Ana06 — CHANGES_REQUESTED** [review]

File: lib/cli.rb (hunk 1 above)
Comment (Ronaq13) [reply lib/cli.rb]: Please help identifying the line at which it modifying the .`yml` file. And by .`yml` file do you mean burndown-data-01.yml file which is created at time of initialization

Comment (Ana06) [reply lib/cli.rb]: inside of the `update` method.

Yes, the `burndown-data-i.yaml` file, where i is the number of the sprint

Comment (Ronaq13) [reply lib/cli.rb]: @Ana06 I dont see any change in the yml file. Please check now is it good to merge?

**Comment by Ana06** [comment]
@Ronaq13 @matheussbernardo @cornelius where are those commits coming from?

**Comment by Ana06** [comment]
@Ronaq13 what is the state of this PR? :thinking:

**Comment by Ana06** [comment]
Closing due to inactivity, @Ronaq13 feel free to reopen it if you continue working on it :wink:

**Pull request closed without merging**